classes/submission/PKPAuthorDAO.inc.php

```diff
@@ -362,8 +361,8 @@ function getPrimaryContact($submissionId) {
 
 	/**
 	 * Remove other primary contacts from a submission and set to authorId
-	 * @param $authorId int
-	 * @param $submissionId int
+	 * @param $authorId int Author ID.
+	 * @param $submissionId int Submission ID.
```

Comment:
Are you adding those comments because of Doxygen? I mean, for us humans :) I think that's kind of duplicated compared to the parameter name...